contrib/test_jenkins.sh

```diff
@@ -460,8 +460,8 @@ run_client_server() {
     UCX_NET_DEVICES=${dev} UCX_TLS=rc ./${test_name} -a ${server_ip} -p ${server_port} &
     hw_client_pid=$!
 
-    wait ${hw_client_pid} ${hw_server_pid}
-
+    wait ${hw_client_pid}
+    kill -9 ${hw_server_pid}
```

Comment:
Can we avoid the kill ? It is kinda bad practice that will lead to zombie processes

Reply:
@shamisp this PR keeps the server alive until the user kills it. Is there a better way to stop the server's PID in Jenkins?

Reply:
@alex--m 
in the current scheme we don't support disconnect in client/server mode so the server has to run forever until it's killed
should not create a zombie because the test does not fork

Reply:
@shamisp is this good to go?

Reply:
ok